lib/pbench/server/api/resources/users_api.py

```diff
@@ -471,17 +471,19 @@ def delete(self, username):
 
         try:
             User.delete(username)
-            self.logger.info("User entry deleted for user with username {}", username)
+            self.logger.info(
+                "User entry deleted for user with username: {}, by user: {}",
```

Comment:
Is this a log entry that will be emitted with all the other logs from server code?  Or is this intended for the CLI interface?  If the former, great, if the latter, then do we really want CLI interface concerns in the core library calls?

Reply:
It is the former. CLI does not involve the authentication, this log is from the API call